### PR TITLE
implement greprintsigns

### PR DESCRIPTION
**Pull request by eroux**
https://github.com/gregorio-project/gregorio/pull/938 but on release-4.1

### CODE REVIEW

File: doc/Command_Index_User.tex [hunk 1]
```diff
@@ -936,6 +936,22 @@ \subsubsection{Bar spacing}
   & \texttt{old} & Activates the old behavior (Default)\\
 \end{argtable}
 
+\subsubsection{Sign printing}
+
+\macroname{\textbackslash greprintsigns}{\{\#1\}\{\#2\}}{gregoriotex-signs.tex}
+Macro to prevent rythmic signs from printing:
+
+\begin{argtable}
+  \#1 & \texttt{vepisemus} & sets the printing of vertical episema\\
```
Comment (henryso) [inline doc/Command_Index_User.tex]: This should really be vepisema/hepisema.

File: tex/gregoriotex-signs.tex [hunk 2]
```diff
@@ -35,6 +35,43 @@
...
+\gre@disablevepisemusfalse
+\newif\ifgre@disablehepisemus
+\gre@disablehepisemusfalse
+\newif\ifgre@disablemora
+\gre@disablemorafalse
+
+\def\greprintsigns#1#2{%
+  \IfStrEq{#2}{enable}%
+    {%
+      \IfStrEq{#1}{vepisemus}{\global\gre@disablevepisemusfalse}{%
```
Comment (henryso) [inline tex/gregoriotex-signs.tex]: This should really be vepisema/hepisema.

File: doc/Command_Index_User.tex [hunk 3]
```diff
@@ -936,6 +936,22 @@ \subsubsection{Bar spacing}
   & \texttt{old} & Activates the old behavior (Default)\\
 \end{argtable}
 
+\subsubsection{Sign printing}
+
+\macroname{\textbackslash greprintsigns}{\{\#1\}\{\#2\}}{gregoriotex-signs.tex}
+Macro to prevent rythmic signs from printing:
```
Comment (henryso) [inline doc/Command_Index_User.tex]: It should probably be noted here that all are enabled by default.

**Comment by henryso** [comment]
This does seem "safe" enough to include in 4.1.  Other than my comments above, I'm OK with this.

**Comment by eroux** [comment]
Thanks! comments are addressed

**Pull request merged**